packages/experimental-identity-and-auth/src/IdentityProviderConfig.ts

```diff
@@ -1,32 +1,26 @@
 import { Identity, IdentityProvider } from "@smithy/types";
 
-import { HttpAuthScheme, HttpAuthSchemeId } from "./HttpAuthScheme";
+import { HttpAuthSchemeId } from "./HttpAuthScheme";
 
 /**
  * Interface to get an IdentityProvider for a specified HttpAuthScheme
  * @internal
  */
-export interface IdentityProviderConfiguration {
+export interface IdentityProviderConfig {
   /**
    * Get the IdentityProvider for a specified HttpAuthScheme.
    * @param schemeId schemeId of the HttpAuthScheme
    * @returns IdentityProvider or undefined if HttpAuthScheme is not found
    */
   getIdentityProvider(schemeId: HttpAuthSchemeId): IdentityProvider<Identity> | undefined;
-
-  /**
-   * Gets the configured HttpAuthSchemes.
-   * @returns all configured HttpAuthSchemes
-   */
-  getAuthSchemes(): Map<HttpAuthSchemeId, HttpAuthScheme>;
 }
 
 /**
  * Default implementation of IdentityProviderConfiguration
  * @internal
  */
-export class DefaultIdentityProviderConfiguration implements IdentityProviderConfiguration {
-  private authSchemes: Map<HttpAuthSchemeId, HttpAuthScheme> = new Map();
+export class DefaultIdentityProviderConfig implements IdentityProviderConfig {
+  private authSchemes: Map<HttpAuthSchemeId, IdentityProvider<Identity>> = new Map();
 
   /**
    * Creates an IdentityProviderConfiguration with a list of HttpAuthSchemes.
```

Comment:
This doc and a few others need to be updated

Reply:
Updated the relevant TS docs in the newest revision.